agents/official/bee-agent-framework/src/server.ts

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env npx -y tsx
+#!/usr/bin/env npx -y tsx@latest
```

Comment:
This shebang will only work on macOS.

macOS parses it as: `["/usr/bin/env", "npx", "-y", "tsx@latest"]`

Linux parses it as: `["/usr/bin/env", "npx -y tsx@latest"]`, but that does not work, since it searches for an executable called `npx -y tsx@latest`.

Fix by using `-S` (pseudo-option in `env` that mitigates this):

```suggestion
#!/usr/bin/env -S npx -y tsx@latest
```